x-pack/docs/en/security/troubleshooting.asciidoc

```diff
@@ -698,29 +698,30 @@ the `basic` `authProvider` in {kib}. The process is documented in the
 
 *Logging:*
 
-Very detailed trace logging can be enabled specifically for the SAML realm by
+Very detailed debug logging can be enabled specifically for the SAML realm by
```

Comment:
```suggestion
If the issue you encounter is not covered in the list above, it might be helpful to enable additional logging for the SAML realm to further troubleshoot the issue. You can enable debug logging by
```

Reply:
Thanks for this suggestion @jkakavas -- I incorporated a version of that text into my changes.